app/models/card.py

```diff
@@ -6,14 +6,17 @@ class Card(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
     text: Mapped[str] = mapped_column(nullable=False)
     likes: Mapped[int] = mapped_column(default=0)
-    board_id = db.Column(Integer, ForeignKey('board.id'), nullable=True)
-    board = relationship('Board', back_populates='cards')
+    # board_id = db.Column(Integer, ForeignKey('board.id'), nullable=True)
+    # board = relationship('Board', back_populates='cards')
+    board_id: Mapped[int] = mapped_column(ForeignKey("board.id"))
+    board: Mapped["Board"] = relationship(back_populates="cards")
```

Comment:
Why is this even different?

Reply:
The "db.Column" syntax is becoming outdated and when I was working on a previous API Kelsey told me we should use the Mapped syntax.